prototype_source/quantization_2_0_tutotial.rst

```diff
@@ -0,0 +1,595 @@
+(prototype) PyTorch Quantization 2.0 Tutorial
+==========================================
+
+Today we have `FX Graph Mode
+Quantization <https://pytorch.org/docs/stable/quantization.html#prototype-fx-graph-mode-quantization>`__
+which uses symbolic_trace to capture the model into a graph, and then
+perform quantization transformations on top of the captured model. In a
+similar way, for Quantization 2.0 flow, we will now use the PT2 Export
+workflow to capture the model into a graph, and perform quantizations
+transformations on top of the ATen dialect graph. This is expected to
+have significantly higher model coverage, better programmability, and
+a simplified UX.
+
```

Comment:
I would add prerequisites here. Something like this

Prerequisites
-------------

This tutorial requires the following:
- Understanding of quantization concepts in PyTorch - link to resource (other tutorials or docs)
- Pytorch 2.0 or later
- ....

Reply:
Thanks for the suggestions. The Prerequisites are added.